python/mcap/mcap/writer.py

```diff
@@ -427,9 +434,13 @@ def __finalize_chunk(self):
 
         chunk_data = self.__chunk_builder.end()
         if self.__compression == CompressionType.LZ4:
+            if lz4 is None:
+                raise RuntimeError("lz4 is not installed")
             compression = "lz4"
-            compressed_data: bytes = lz4.frame.compress(chunk_data)  # type: ignore
```

Comment:
Why did this switch away from frame compression?

Reply:
The import changed: `import lz4.frame as lz4`

Reply:
Right :+1: